src/Charts/Pie/Pie.js

```diff
@@ -0,0 +1,172 @@
+import React, { Component } from 'react';
+import propTypes from 'prop-types';
+import { generate } from 'c3';
+import { select } from 'd3';
+import classNames from 'classnames';
+import { Link } from 'react-router-dom';
+import isEqual from 'lodash/isEqual';
+
+import './pie.scss';
+
+export const LegendPosition = {
```

Comment:
No need to define such const here, you can import `LegendPosition` from Donut chart.